dashboard/src/main/resources/jenkins.properties

```diff
@@ -0,0 +1,5 @@
+jenkins.url=http://jenkins
+
+jenkins.login=xpaasqe-admin
+jenkins.apiToken=3bf6ab02b4928a54c043cbb14483199c
```

Comment:
Don't publish secrets! Delete this file from the PR and if this is real token, contact someone to invalidate it right away.